silkworm/rpc/http/request_handler.cpp

```diff
@@ -57,19 +56,18 @@ Task<void> RequestHandler::handle(const std::string& content) {
             if (!is_valid_jsonrpc(item.value())) {
                 batch_reply_content << make_json_error(0, -32600, "invalid request").dump();
             } else {
-                Channel::Response single_reply;
+                std::string single_reply;
                 send_reply = co_await handle_request_and_create_reply(item.value(), single_reply);
-                batch_reply_content << single_reply.content;
+                batch_reply_content << single_reply;
             }
         }
         batch_reply_content << "]\n";
 
-        msg_response.status = Channel::ResponseStatus::ok;
-        msg_response.content = batch_reply_content.str();
+        response = batch_reply_content.str();
     }
 
     if (send_reply) {
-        co_await channel_->write_rsp(msg_response);
+        co_await channel_->write_rsp(response);
```

Comment:
I suppose you can move `response` here if `Channel::write_rsp` accepts a r-value parameter

Reply:
ok